### PR TITLE
[MM-35039] - Send trial ended email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Adds ability to listen to trial ended webhook from the CWS and then send a trial ended email to all admins
Screenshot 2021-04-21 at 23 19 18

#### Ticket Link
[MM-35039](https://mattermost.atlassian.net/browse/MM-35039?atlOrigin=eyJpIjoiNDgyYjg0Nzg5NzAwNGM1NzhiMmU3ODU5YTkzMmE3MWUiLCJwIjoiaiJ9)

#### Related Pull Requests
- Has webapp changes (https://github.com/mattermost/mattermost-webapp/pull/7967)

#### Release Note
```release-note
None
```

### CODE REVIEW

**Review by stevemudie — APPROVED** [review]
Will test after merge.

File: app/cloud.go [hunk 1]
```diff
@@ -200,3 +204,33 @@ func (a *App) SendCloudTrialEndWarningEmail(trialEndDate, siteURL string) *model
 
 	return nil
 }
+
+func (a *App) SendCloudTrialEndedEmail() *model.AppError {
+	sysAdmins, e := a.getSysAdminsEmailRecipients()
+	if e != nil {
+		return e
+	}
```
Comment (marianunez) [inline app/cloud.go:212]: `err` as naming convention

```suggestion
	sysAdmins, err := a.getSysAdminsEmailRecipients()
	if err != nil {
		return err
	}
```

**Review by marianunez — APPROVED** [review]
LGTM @AGMETEOR!

**Comment by AGMETEOR** [comment]
Hey @nickmisasi gentle ping

**Review by nickmisasi — APPROVED** [review]

**Comment by AGMETEOR** [comment]
Hey @annelieseklein , just adding you as a final reviewer for the template of this email. You can see the image in the description above.

**Review by annelieseklein — APPROVED** [review]

**Pull request merged**